### PR TITLE
Feat/incorporations refactor

### PR DESCRIPTION
**Pull request by maycon-mello**
Refactor on incorporations marketplace
Created storybook stories for incorporations
Extracted common components to /marketplace/common

### CODE REVIEW

**Comment by andregoncalves** [comment]
@maycon-mello great work.

Can you please take a look at the changes in BAM components in this branch: feat/redirect-to-kycc and apply the same changes to incorporations.
More specifically, there is now a MarketplaceComponent base class that encapsulates some common logic for routing and checking KYC process status.
Thanks, sorry, this is a such moving target.

**Comment by maycon-mello** [comment]
Sure @andregoncalves. I'm working on it now. Thanks

**Comment by maycon-mello** [comment]
This is ready for a new review

**Comment by andregoncalves** [comment]
@maycon-mello I'm waiting for #1357 to be merged before I do a full review.

**Comment by maycon-mello** [comment]
@andregoncalves i've fixed the merge conflicts

File: src/renderer/marketplace/incorporation/checkout/checkout-container.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import { kycOperations } from 'common/kyc';
+import { incorporationsSelectors } from 'common/incorporations';
+import { withStyles } from '@material-ui/core/styles';
+import { getIncorporationPrice } from '../common';
+import { ethGasStationInfoOperations } from 'common/eth-gas-station';
+import PaymentCheckout from '../../common/payment-checkout';
+import * as CheckoutUtil from '../../common/checkout-util';
```
Comment (andregoncalves) [inline src/renderer/marketplace/incorporation/checkout/checkout-container.jsx]: this functionality available in the helper, should use marketplace common component instead, right?

Comment (maycon-mello) [reply src/renderer/marketplace/incorporation/checkout/checkout-container.jsx]: Yes, i thought I did this refactor in the last changes, maybe it got lost when solving merge conflicts, my bad. I've created a new commit with the changes. Now, incorporations checkout will use the marketplace commons components and CheckoutUtil.js got removed from the wallet

**Comment by maycon-mello** [comment]
@andregoncalves i've fixed the merge conflicts

**Comment by andregoncalves** [comment]
This work has been integrated into another branch: #1471 .
Closing the pull request and branch.

**Pull request closed without merging**